test/integration/targets/podman_image_info/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+- name: Test podman_image_info
+  when:
+    - ansible_facts.virtualization_type != 'docker'
```

Comment:
Similar comment as below about podman availability, unless this is being filtered by aliases in a way I'm not familiar with.